op-supervisor/config/config.go

```diff
@@ -25,10 +27,20 @@ type Config struct {
 	// MockRun runs the service with a mock backend
 	MockRun bool
 
-	L2RPCs  []string
+	// TODO(protocol-quest#288): configure list of chains and their RPC endpoints / potential alternative data sources
```

Comment:
> from the pr description

*Alternative Design*
a) Recommended. This is what I think the code should do in long term. but I didn't go with this as it is a rather big change that changes the interface of cli arguments.

Rationale: Rather than having a global config for all L2 chain monitors, I think each should have its own RPC config that sets to default value unless overridden.

```
&Config{
	L2RPCConfigs:        [
	        L2RPCConfig {
		        Url: "http-some-alchemchy-url",
		        EpochPollInterval: 30 * time.Second,
		        PollInterval:      2 * time.Second,
		        ShouldTrustRpc:    false,
		        RpcKind:           sources.RPCKindStandard,
	        },
	        L2RPCConfig{
	                // use default if not filled
	                Url:  "some other url",
	        }
	]
}
```

Reply:
Let me know if you want me to proceed with this design here.